.github/ci-init.sh

```diff
@@ -0,0 +1,12 @@
+echo "Installing Quarkus CLI"
+find "${HOME}/.m2/repository/io/quarkus/quarkus-cli/" -name "quarkus-cli*runner.jar" -printf "%T+|%p\n" |
+  sort |
+  tail -n 1 |
+  cut -d '|' -f 2 |
+  xargs jbang app install --force --name quarkus
```

Comment:
Why not just `jbang app install --force --name quarkus io.quarkus:quarkus-cli:999-SNAPSHOT:runner`?

Reply:
I wanted to use the freshly built version, like we do in our CI. And it might not always be `999-SNAPSHOT`. In our case, it is for now. But we may, for some reason, decide to build a release tag.

Reply:
Ecosystem CI is always against 999-SNAPSHOT

Reply:
Ok, but since we do not control the version, I used a variable passed from calling script, rather than hard-coded value:
```
jbang app install --force --name quarkus "io.quarkus:quarkus-cli:${QUARKUS_VERSION}:runner"
```